backend/src/backend/main.py

```diff
@@ -68,6 +70,11 @@ async def test(timestamp: int):
 async def add_book(book_payload: BookAddSchema):
     """API endpoint to receive Book addition requests and add to database"""
 
+    ident = get_ident_from_name(book_payload.metadata["Name"])
+    if ident:
+        # if valid ident
+        title = await Title.objects.get_or_create(ident=ident)
+
```

Comment:
We need to fill the Title with data as well. Can be done after the book is created though.